docs/content/how-to/openstack/create-openstack-cluster.md

```diff
@@ -0,0 +1,282 @@
+
+## Installing HyperShift Operator and cli tooling
+
+Before creating a guest cluster, the hcp cli, hypershift cli, and HyperShift
+Operator must be installed.
+
+The `hypershift` cli tool is a development tool that is used to install
+developer builds of the HyperShift Operator.
+
+The `hcp` cli tool is used to manage the creation and destruction of guest
+clusters.
+
+### Build the HyperShift and HCP CLI
+
+The command below builds latest hypershift and hcp cli tools from source and
+places the cli tool within the /usr/local/bin directory.
+
+!!! note
+
+    The command below is the same if you use docker.
+  
+```shell
+podman run --rm --privileged -it -v \
+$PWD:/output docker.io/library/golang:1.22 /bin/bash -c \
+'git clone https://github.com/openshift/hypershift.git && \
+cd hypershift/ && \
+make hypershift product-cli && \
+mv bin/hypershift /output/hypershift && \
+mv bin/hcp /output/hcp'
+
+sudo install -m 0755 -o root -g root $PWD/hypershift /usr/local/bin/hypershift
+sudo install -m 0755 -o root -g root $PWD/hcp /usr/local/bin/hcp
+rm $PWD/hypershift
+rm $PWD/hcp
+```
+
+## Deploy the HyperShift Operator
+
+Use the hypershift cli tool to install the HyperShift operator into the
+management cluster.
+
+```shell
+hypershift install
+```
+
+You will see the operator running in the `hypershift` namespace:
+
+```shell
+oc -n hypershift get pods
+
+NAME                        READY   STATUS    RESTARTS   AGE
+operator-755d587f44-lrtrq   1/1     Running   0          114s
+operator-755d587f44-qj6pz   1/1     Running   0          114s
+```
+
+## Upload RHCOS image in OpenStack
+
+For now, we need to manually push an RHCOS image that will be used when deploying the node pools
+on OpenStack. In the future, the CAPI provider (CAPO) will handle the RHCOS image lifecycle by using
+the image available in the chosen release payload.
+
+Here is an example of how to upload an RHCOS image to OpenStack:
+
+```shell
+openstack image create --disk-format qcow2 --file rhcos-417.94.202407080309-0-openstack.x86_64.qcow2 rhcos-4.17
+```
+
+## Create a HostedCluster
+
+Once all the [prerequisites](#prerequisites) are met, and the HyperShift
+operator is installed, it is now possible to create a guest cluster.
+
+Below is an example of how to create a guest cluster using environment
+variables and the `hcp` cli tool.
+
+!!! note
+
+    The --release-image flag could be used to provision the HostedCluster with a specific OpenShift Release (the hypershift operator has a support matrix of releases supported by a given version of the operator)
+
+
+```shell linenums="1"
+export CLUSTER_NAME=example
+export PULL_SECRET="$HOME/pull-secret"
+export WORKER_COUNT="2"
+
+# clouds.yaml should be in the format of the OpenStack clouds.yaml file and the cloud name has to be 'openstack' for now.
+export CLOUDS_YAML="$HOME/clouds.yaml"
+
+# Image name is the name of the image in OpenStack that was pushed in the previous step.
+export IMAGE_NAME="rhcos"
```

Comment:
nit: The name used here is different from the image name created in the previous step

Reply:
oops, let me correct that.